packages/example/components/chains/babylon/example.tsx

```
@@ -0,0 +1,703 @@
+            );
+          }}
+          onGenerateRequest={async (fromData: Record<string, any>) => {
+            const toAddress = fromData['toAddress'] as string;
+            const amount = parseInt(fromData['amount'] as string);
+            const gasPrice = parseInt((fromData['gasPrice'] as string) ?? '20');
+
+            if (!toAddress || !amount) {
+              throw new Error('toAddress or amount is required');
+            }
+
+            const network = await providerBitcoin?.getNetwork();
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)